pkg/morph/client/container/nodes.go

```diff
@@ -0,0 +1,50 @@
+package container
+
+import (
+	"fmt"
+
+	"github.com/nspcc-dev/neofs-node/pkg/morph/client"
+)
+
+// AddNextEpochNodes registers public keys as a container's placement vector
+// with specified index. Registration must be finished with final
+// [Client.CommitContainerListUpdate] call. Always sends a notary request with
+// Alphabet multi-signature.
+func (c *Client) AddNextEpochNodes(cid []byte, placementIndex int, nodesKeys [][]byte) error {
```

Comment:
no specific idea. since it is a real _go_ code, i wanted indexes to be `int` while REPs in our SDK are `uint32`. i wanted it to be something special but there are no types in neo VM. i even had to change something on the contract side just because it did not compile
about this thread: it was not a mistake, made intentionally, if you do not agree, ping me one more time and i will change